dotnetv3/Aurora/Scenarios/AuroraScenario.cs

```diff
@@ -0,0 +1,563 @@
+
+using Amazon.RDS.Model;
+using Amazon.RDS;
+using AuroraActions;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Console;
+using Microsoft.Extensions.Logging.Debug;
+
+namespace AuroraScenario;
+
+/// <summary>
+/// Scenario for Aurora examples.
+/// </summary>
+public class AuroraScenario
+{
+    /*
+    Before running this .NET code example, set up your development environment, including your credentials.
+
+    This .NET example performs the following tasks:
+    1.  Return a list of the available DB engine families for Aurora MySql using the DescribeDBEngineVersionsAsync method.
+    2.  Select an engine family and create a custom DB cluster parameter group using the CreateDBClusterParameterGroupAsync method.
+    3.  Get the parameter group using the DescribeDBClusterParameterGroupsAsync method.
+    4.  Get some parameters in the group using the DescribeDBClusterParametersAsync method.
+    5.  Parse and display some parameters in the group.
+    6.  Modify the auto_increment_offset and auto_increment_increment parameters
+        using the ModifyDBClusterParameterGroupAsync method.
+    7.  Get and display the updated parameters using the DescribeDBClusterParametersAsync method with a source of "user".
+    8.  Get a list of allowed engine versions using the DescribeDBEngineVersionsAsync method.
+    9.  Create an Aurora DB cluster that contains a MySql database and uses the parameter group 
+        using the CreateDBClusterAsync method.
+    10. Wait for the DB cluster to be ready using the DescribeDBClustersAsync method.
+    11. Display and select from a list of instance classes available for the selected engine and version
+        using the paginated DescribeOrderableDBInstanceOptions method.
+    12. Create a database instance in the cluster using the CreateDBInstanceAsync method.
+    13. Wait for the DB instance to be ready using the DescribeDBInstances method.
+    14. Display the connection endpoint string for the new DB cluster.
+    15. Create a snapshot of the DB cluster using the CreateDBClusterSnapshotAsync method.
+    16. Wait for DB snapshot to be ready using the DescribeDBClusterSnapshotsAsync method.
+    17. Delete the DB instance using the DeleteDBInstanceAsync method.
+    18. Delete the DB cluster using the DeleteDBClusterAsync method.
+    19. Wait for DB cluster to be deleted using the DescribeDBClustersAsync methods.
+    20. Delete the cluster parameter group using the DeleteDBClusterParameterGroupAsync.
+    */
+
+    private static readonly string sepBar = new('-', 80);
+    private static AuroraWrapper auroraWrapper = null!;
+    private static ILogger logger = null!;
+    private static readonly string engine = "aurora-mysql";
+    static async Task Main(string[] args)
+    {
+        // Set up dependency injection for the Amazon RDS service.
```

Comment:
the Amazon RDS service. --> Amazon Relational Database Service (Amazon RDS).

Reply:
Fixed.